CorsixTH/Lua/world.lua

```diff
@@ -1242,7 +1242,7 @@ end
 -- during the coming month.
 function World:updateSpawnDates()
   -- Set dates when people arrive
-  local no_of_spawns = math.n_random(self.spawn_rate, 2)
+  local no_of_spawns = math.n_random(self.spawn_rate * self:getReputationImpact(self:getLocalPlayerHospital()), 2)
```

Comment:
Agreed, there is reference in the original theme hospital to a world spawn rate and a pie chart of what is allocated to each hospital, we should do our hospital's spawn there.